<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VmapViewer</class>
 <widget class="QWidget" name="VmapViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="heightButton">
         <property name="text">
          <string>Height</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="metaButton">
         <property name="text">
          <string>Meta</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Masks</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="maskCombo">
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Delta</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ObjShadow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Material</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Double Level</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Shadowed</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
